### DISCUSSION TITLE
Is it possible to auto-generate changeset with Renovate?

### DISCUSSION THREAD

**Discussion by urrumirko** [renovatebot/renovate#24060] [Request Help]
### How are you running Renovate?

Placeholder value, please select the correct response from the dropdown

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

Hello everybody

I've been using Renovate for a while and I'm starting to face a little issue: I don't have track of these changes in the changelog.

I don't know if this is possible, but I was wondering if there is a way to generate a changelog (with changeset) during renovate MR generation.

Thanks in advance however will answer

Cheers

### What you tried so far.

I was looking around for pre-commit hook but I didn't find any reference to changeset

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by astellingwerf** [comment]
I'm not aware of an OOTB solution for this, but you could possibly get something to work with [`postUpgradeTasks`](https://docs.renovatebot.com/configuration-options/#postupgradetasks), combined with [templates](https://docs.renovatebot.com/templates/).
